src/foam/u2/view/ScrollTableView.js

```diff
@@ -190,7 +227,44 @@
       class: 'Boolean',
       name: 'isInit'
     },
-    'tableWrapper_'
+    'tableWrapper_',
+
+    // Navigation props
+    {
+      class: 'Boolean',
+      name: 'showNav',
```

Comment:
Maybe showPagination or enablePagination would be more explicit and easier to understand since there could be multiple forms of navigation